lib/mugshot.js

```diff
@@ -276,10 +276,11 @@ function _test(baseName, options, screenshot, done) {
           return done(error);
         }
 
-        done(null, {
-          isEqual: options.acceptFirstBaseline,
-          baseline: baselinePath
-        });
+        if (options.acceptFirstBaseline) {
+          done(null, {isEqual: true, baseline: baselinePath});
+        } else {
+          done(new Error('Baseline missing'));
```

Comment:
The convention is that when an error occurs you can't expect any useful/relevant information in the result object so it's ok to just not return it.